README.md

```diff
@@ -7,9 +7,29 @@ This repository contains the source of the https://docs.fyne.io website.
 The website uses [jekyll](https://jekyllrb.com/) to generate the website from markdown files.
 With ruby installed it is as easy as running the following commands in a terminal (the first only needs to run once):
 
+### Install jekyll
 ```bash
 gem install bundler jekyll
-bundle exec jekyll serve
+```
+
+### Clone the Fyne docs repository
+```bash
+git clone https://github.com/fyne-io/docs.fyne.io.git
+```
+
+### Change directory in to the repository
```

Comment:
Does this need to be in a new section? It feels like cd after git clone could be all under the Clone header

Reply:
Thats true :)

Reply:
Sorry i am fairly new to git (still learning). Should i make a new PR?